_posts/2017-02-07-women-technology-careers.md

```diff
@@ -0,0 +1,22 @@
+
+Now, I’m fortunate and not fortunate equally at the same time. I went back to my degree as a mature student with children, I therefore possessed life skills and confidence from literally ‘living’. However, I originally trained as a dental nurse, while behind closed doors I would code and code and code… never did I tell anyone though, honestly even if I did, like when I discuss coding now, no one has a bloody clue or remote interest in what I do unless it involves infamous discussions like Facebook and hacking. But I don’t mind, in fact it has it’s perks, when you’re in general discussions outside work and people ask you “So, what do you do?” With already 90% of the people who ask have it already imprinted on their minds that obviously with three children I am a stay at home mum. When I reply with “I’m a Software Engineer” my do I get some confused looks, it’s not their fault though, 60% haven’t even got a clue what my actual job role is and the remaining 40% usually look at me like some form of alien thats abandoned their children. However, becoming a ‘woman’ in technology has it’s perks. In my research numerous amounts of women that I know have gone into teaching simply for the term time hours. They’re filled with the idea that they are able to reduce their childcare bill dramatically, still have sufficient time with their children, yet still have the ability to be a working mother without to much impact on their family life. Don’t believe the dream. Term-time roles are excellent for the prevention of huge or extended childcare bills, however teachers don’t work 9-3, or the ones I discuss this with don’t. Teaching is a long-hour daily role and should not be under-estimated, teachers work hard and have high expectations and masses of responsibility for the development and education of our future generation. However, work in IT and actually have the ability to earn a usually very well paid role and work remotely (from home) numerous times (or sometimes full time) all week. Obtain benefits and discounts to help your family including many professional benefits such as private healthcare and childcare vouchers, obtain these with being able to also do school runs, fly to those doctor appointments, have the ability to stay off with your sick child and don’t feel as though you are going to lose your job! Don’t feel punished for being a working mum! If you yourself now, yes you who is reading this, search for an IT related role in Indeed for example, right now with the location as remote you will see the mass demand this sector has, not just for women but for people who have the tech skills or who are willing to learn.
```

Comment:
Consider changing towards the end of this first phase to:

... "never did I tell anyone, though: honestly, even if I did, no one I know had a bloody clue or remote interest in what I was talking about unless it was involving something like Facebook or hacking — this is still the case now I do it full-time."

"its perks" rather than "it's perks" - and switch the comma out to a colon.

Although "With already 90% of the people who ask it" sounds humourous, the figure is anecdotal and you can get the same effect without the arbitrary figure: something like "When they see I have three children, a lot of people have it imprinted on their mind that I'm **obviously** a stay-at-home mum, and when I reply with, "I'm a software engineer", my, do I get some confused looks!   It's not their fault though; it's not a job role that most people know what it actually is."

I'd then be tempted to split into another paragraph for the part starting "However, being a woman in technology" (if you want to have quotes, I'd wrap it around the whole of 'woman in technology' as it feels like it's that part you're trying to emphasise rather than calling the status of being a woman into question): also, it should be "its perks", not "it's perks".

"In my research" and "numerous women that I know" are a bit contradictory - either go with "In my research, many women have gone..." or "Numerous women that I know have gone..."

"Term-time", not "term time".

"too much impact", not "to much impact".

Again, have commas around the "however".

I haven't heard the phrase "long-hour" before - that's not to say it's not a thing, but it doesn't seem to fit.  How about, "demanding", instead?

"underestimated" is one word, shouldn't be hyphenated.

Feels like there's a word missing in "work in IT" - maybe, "However, if you work in IT, you actually have..."

You can't earn a role: perhaps go with something like, "secure a usually very well-paid role".

"well-paid" should be hyphenated; so should "full-time" - but it's not "all week", perhaps "each week" or "every week"?

Try reworking the next sentence as, "Many businesses offer benefits and discounts to help your family, including many professional benefits such as private healthcare and childcare vouchers: it is possible in a good company for you to still do the school run or fly off to those doctors' appointments that crop up, or to take time off to look after your sick children without ever feeling like you are going to lose your job!".

Certainly, not all employers in tech do offer these benefits or have this level of understanding, I'm afraid to say.

"IT-related" should be hyphenated.

"on Indeed", not "in Indeed" - and maybe a link to Indeed?